make.sh

```diff
@@ -97,3 +97,10 @@ line_number=$(grep -n -i "# Development" podfile | cut -f1 -d:)
 sed -i.bak "$(($line_number + 1))i\\"$'\n'"\
   pod 'shared', :path => '../shared'\\
 " podfile
+
+# Correct path in Dangerfile
+sed -i '' "s/require '\.\/fastlane\/Constants\/Constants'//g" dangerfile
+sed -i '' "s/'.\//'.\/ios\//g" dangerfile
+sed -i '' 's/".\//".\/ios\//g' dangerfile
```

Comment:
Does this line and the above line have a similar job?

Reply:
One deal with `'.`, one deal with `".` 🤣